knowledge-base/adr/0031-resilience-options.md

```diff
@@ -49,36 +49,36 @@ executeHttpRequest(
 
 So we have something for circuit breaker and timeout.
 
-|     Option      | On target | On BTP | Default target | Default BTP |
-| :-------------: | :-------: | :----: | :------------: | :---------: |
-| circuit breaker |    ❌     |   ✅   |      n.a.      |   enabled   |
-|     timeout     |    ✅     |   ✅   |    enabled     |   enabled   |
-|      retry      |    ❌     |   ❌   |      n.a.      |    n.a.     |
-|   rate limit    |    ❌     |   ❌   |      n.a.      |    n.a.     |
-|   bulk limit    |    ❌     |   ❌   |      n.a.      |    n.a.     |
+|     Option      | On target<br/>avail/default | On BTP<br/>avail/default |   Remarks   |
+| :-------------: | :-------------------------: | :----------------------: | :---------: |
+| circuit breaker |            ❌/❌            |          ✅/✅           | via options |
+|     timeout     |            ✅/✅            |          ✅/✅           | via options |
+|      retry      |            ❌/❌            |          ❌/❌           |    n.a.     |
 
 ## Decision
 
 - Rate limit and bulk limit we will not be done -> no user request.
-- We will implement an [extendable middleware approach](#api).
-- Per default the resilience middleware is off.
+- We will implement an [extendable middleware approach](#api) to add resilience to the target system.
+- Per default there is no (resilience) middleware.
+- There is circuit breaker and timeout on BTP services which is on and can not be configured.
 - This is a breaking change and therefore will be included in a future major version upgrade.
-- All resilience related options (timeout, circuitbreaker) are deprecated in version 2.
+- All resilience related options (timeout, circuit breaker) are removed in version 3.
```

Comment:
[nth] There should be 3 tables and each be labeled accordingly: "Pre-ADR", "Post-ADR", and "v3" (the last one would be additional to those that are in here now).